docs/admin/upgrade_kytos.rst

```diff
@@ -1,3 +1,44 @@
 ********************
-How to upgrade kytos
+How to upgrade Kytos
 ********************
+
+Before upgrading Kytos, we recommend backing up the kytos settings and the
+NApps directory. To do this, follow the instructions.
+
+Backup
+======
+
+Run the following command to backup the files:
+
+.. code-block:: bash
+
+  $ tar cvfz ~/kytos-backup.tar.gz /etc/kytos /var/lib/kytos
```

Comment:
@beraldoleal this command was my suggestion. Please confirm that this is an acceptable basic backup procedure, or suggest something else.